app/views/admin/users/widgets/_organizations.html.erb

```diff
@@ -56,9 +58,9 @@
       <%= f.text_field :organization_id, class: "crayons-textfield", size: 8, required: true, inputmode: "numeric", type: "number", placeholder: "1234" %>
     </div>
     <div class="crayons-field">
-      <%= f.label "Permission level", class: "crayons-field__label" %>
+      <%= f.label :type_of_user, "Permission level", class: "crayons-field__label" %>
```

Comment:
Changed this to `:type_of_user` here and for the `f.select` so that the label is linked to the combobox.